package/graph.py

```diff
@@ -1,38 +1,65 @@
-import os
 import subprocess
 import sys
 
 
-def display(snippet_name):
-    snippet_path = os.path.join(
-        os.path.dirname(__file__), "stash", f"{snippet_name}.py"
-    )
+def display(snippet_name, source_code):
+    """
+    Display the snippet's content and copy it to the clipboard.
 
-    with open(snippet_path, "r") as file:
-        source_code = file.read()
+    Args:
+        snippet_name (str): The name of the snippet.
+        source_code (str): The source code to be copied and displayed.
+    """
 
-    # Function to copy text to clipboard based on platform
     def copy_to_clipboard(text):
-        # linux
-        if "linux" in sys.platform:
-            subprocess.run(
-                ["/usr/bin/xclip", "-selection", "clipboard"],
-                input=text.strip().encode(),
-                check=True,
-            )
-        # windows
-        elif "win32" in sys.platform:
-            subprocess.run(
-                ["C:\\Windows\\System32\\clip.exe"],
-                input=text.strip().encode(),
-                check=True,
-            )
-        # macOS
-        elif "darwin" in sys.platform:
-            subprocess.run(
-                ["/usr/bin/pbcopy"],  # Full path to pbcopy for macOS
-                input=text.strip().encode(),
-                check=True,
-            )
-        else:
-            raise OSError("Unsupported operating system")
+        """
+        Copy the given text to the system clipboard.
+
+        Args:
+            text (str): Text to be copied to the clipboard.
+        """
+        try:
+            # Linux
+            if "linux" in sys.platform:
+                subprocess.run(
+                    ["/usr/bin/xclip", "-selection", "clipboard"],
+                    input=text.strip().encode(),
+                    check=True,
+                )
+
+            # Windows
+            elif "win32" in sys.platform:
+                subprocess.run(
+                    ["C:\\Windows\\System32\\clip.exe"],
+                    input=text.strip().encode(),
+                    check=True,
+                )
+
+            # macOS
+            elif "darwin" in sys.platform:
+                subprocess.run(
+                    ["/usr/bin/pbcopy"],  # Full path to pbcopy for macOS
+                    input=text.strip().encode(),
+                    check=True,
+                )
+            else:
+                raise OSError("Unsupported operating system")
+
+        except FileNotFoundError:
+            print("Error: Clipboard utility not found. Cannot copy to clipboard.")
+        except subprocess.CalledProcessError as e:
+            print(f"Error during clipboard copying: {e}")
```

Comment:
We don't need obvious comments that would expose the integration process.
Instead, create print statements that resemble the error typically seen in python.